sources/packages/backend/apps/workers/src/controllers/assessment/assessment.dto.ts

```diff
@@ -82,6 +82,14 @@ export interface ApplicationAssessmentJobOutDTO {
    * Application associated with this application.
    */
   applicationId: number;
+  /**
+   * Current status of the application.
+   */
+  applicationStatus: string;
```

Comment:
We can have it defined with the `ApplicationStatus` enum.

Reply:
Updated